client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +49,41 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    @Getter
+    private final boolean isSubscriber;
+    @Getter
+    private final boolean autoTruncateAtLastCheckpoint;
+
    public static class ReaderGroupConfigBuilder implements ObjectBuilder<ReaderGroupConfig> {
        private long groupRefreshTimeMillis = 3000; //default value
        private long automaticCheckpointIntervalMillis = 30000; //default value
        // maximum outstanding checkpoint request that is allowed at any given time.
        private int maxOutstandingCheckpointRequest = 3; //default value
+       private boolean subscribedForRetention = false; //default value
+       private boolean autoPublishLastCheckpoint = false; //default value
+
+       /**
+        * Makes the reader group a subscriber reader group. During CBR (Consumption Based Retention),
+        * the subscriber reader group's positions are considered during truncation. The stream will truncate
+        * to the last checkpoint position of this group.
+        *
+        * @return Reader group config builder.
+        */
+       public ReaderGroupConfigBuilder isSubscribedForRetention() {
+           this.subscribedForRetention = true;
```

Comment:
Default value for "isSubscriber" is "false". 
Please change this method name to `subscribeForRetention` and inside you can set the value to `true` as is done here....

Reply:
The builder class puts `subscribedForRetention` as false by default and `subscribedForRetention` is used in the constructor for `ReaderGroupConfig` so the default value will always be false. The method name will be changed to `subscribeForRetention`.

Reply:
Method name has been changed. Please have a look.